core/node/shared/src/genesis.rs

```diff
@@ -186,6 +184,44 @@ pub struct GenesisBatchParams {
     pub rollup_last_leaf_index: u64,
 }
 
+// TODO: add metrics to track latency of each stage
```

Comment:
Well, so (here)[https://github.com/matter-labs/zksync-era/blob/31db44f31316d1e838f87e6f8cf9412ce0d6c9e4/core/lib/zksync_core/src/metadata_calculator/helpers.rs#L436] we also record some metrics. Logically, they have to be here as well...but then there are two ways:
1) move `LoadChangesStage` somewhere else.
2) create another `genesis` metrics with a similar purpose(I do not like this idea)

Reply:
I believe it's acceptable to not track metrics for the genesis batch; it's a one-time action.